src/Apps/Legion/Components/Diagram/Diagram.types.ts

```diff
@@ -0,0 +1,22 @@
+import { IViewEntity } from '../../Models';
+
+export interface IDiagramProps {
+    parentRef: React.MutableRefObject<any>;
```

Comment:
what does the ref give us if we don't know the type of hte ref? Can we not have `any` here?

Reply:
It is referring to the container parent element which is HTMLDivElement in our use case. I am adding it now.